.travis.yml

```diff
@@ -54,6 +54,9 @@ matrix:
   - os: osx
     osx_image: xcode8.3
     language: generic
+  - os: osx
+    osx_image: xcode9
```

Comment:
This is the new environment added as part of e2167d787a91cfd9d147d0a4bd7bb37ba40f626e